packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -113,8 +113,9 @@ export interface SliderCommons {
    *
    * - If `true`, marks are visible at each `step`.
    * - If `number[]`, marks will be displayed at each provided number. Numbers must be in ascending order.
+   * - If `{}[]` A mark is shown at the value location and displays any provided custom labels and marks.
    */
-  marks?: boolean | number[];
+  marks?: boolean | number[] | (number | { value: number; label?: string | JSX.Element; mark?: JSX.Element })[];
```

Comment:
I think this could just be:

```suggestion
  marks?: boolean | (number | { value: number; label?: string | JSX.Element; mark?: JSX.Element })[];
```